File: C#/06-Condicional If/Condif.cs
using System;

class Condif 
{
    static void Main(string[] args)
    {
     //Vamos utilizar o condicional IF  simples para testar uma condição
        string nome;

        Console.Write("Digite seu nome : ");
        nome = Console.ReadLine();

        Console.WriteLine("Escolha uma porta : 1, 2 ou 3");
        string userValue = Console.ReadLine();
        //foi definido uma string que armazena comando de entrada

        //Vamos testar a condição(Se o usuario digitar 1)
        if(userValue == "1")
        {
            Console.WriteLine("Parabéns " +nome +" você ganhou um carro!");
            Console.ReadLine();
        }
/* Se o usario digitar qualquer outra coisa diferente de 1 bloco de código acima não sera executado
 e o programa encerra porqu não tem mais código*/
    }
}

File: Python/09-Condicional If/CondIf.py
#O condicional if testa uma condição e se e somente se ela for verdadeira o bloco é executado

nome = input('Digite seu nome : ').title()
print('Seja bem vindo {0}'.format(nome))

porta = int(input('1 , 2 ou 3: Escolha uma porta : '))

#Este bloco só será executado se o usuario digitar 1
if porta == 1 :
    print('Parabéns, você ganhou um carro!')

#Este comando será executado independente do que o usuario digitar(mesmo que bloco do if seja executado)
print('Jogo encerrado')
